format_spec/FORMAT_SPEC.md

```diff
@@ -4,7 +4,7 @@ title: Format Specification
 
 **Notes:**
 
-* The current TileDB format version number is **18** (`uint32_t`).
+* The current TileDB format version number is **20** (`uint32_t`).
```

Comment:
Why not 19?

Reply:
Heads up, this will be a merge issue with the Enumerations PR I'm getting ready to merge. But I guess I get to run CI again first since I haven't added this line to that PR.